### ISSUE TITLE
./components/Rectangle.js

### ISSUE THREAD

**Issue by thierryc** [inlet/react-pixi#66]
### Description

I try to do a custom components. 
I get an error: 
Attempted to assign to readonly property.

### Steps to reproduce

1. I have juste add your example. (has is)  to a simple React project. 
2. I add it to the stage.

### Additional info

- `@inlet/react-pixi` version: "^0.3.0",
- `React` version: "^16.6.1"
- `ReactDOM` version: "^16.6.1"
- `PIXI` version: "^4.8.2"

**Comment by inlet** [comment]
Can you tell me which example? (Reproduce step 1)

You can see a working demo here:
https://codepen.io/inlet/pen/RMLJKm

**Comment by thierryc** [comment]
You can easily add new components to your project:

./components/Rectangle.js

**Comment by inlet** [comment]
Can you create a codepen illustrating the issue? I cannot reproduce it, thanks!

**Comment by thierryc** [comment]
I will do a small project to reproduce it. On Codepen your example works.

**Comment by thierryc** [comment]
https://github.com/thierryc/pixipixi 
On this project I don't get any error, but i don't get the red rectangle under the orange bunny too... 

My current project is to build over Next.js.
I will do another test on a NEXT.js project to (try) reproduce the error.

**Comment by thierryc** [comment]
I create a next.js project: https://github.com/thierryc/nextpixipixi-example
I get 2 different error. (Safari / Chrome)

<img width="1065" alt="image" src="https://user-images.githubusercontent.com/197471/49475747-06988e00-f7e6-11e8-8381-517c20237256.png">

**Comment by thierryc** [comment]
no error if you use a Sprite or a Container.

**Comment by inlet** [comment]
I think something else is causing the issue, if you can isolate it in a simple codepen then we can work on a fix 😉

Thanks!

**Comment by thierryc** [comment]
Yes something (else) is causing the issue.

Codepen project works... but  PixiComponent do not works in a project builds on 2 of the main React frameworks. 

**React app** & **Next,js**

On a very simple React app project, **the rectangle is not rendered.**
On a very simple Next.js, **PixiComponent trigger an error.**  

I suspect something in the PixiComponent, I will explore... do you have any advice ?

**Comment by inlet** [comment]
I think the problem is not React Pixi, but rendering the canvas in a node process (just a wild guess though), since nextjs is basically an express app.

You can try wrapping the pixi stuff in a [NoSSR](https://github.com/kadirahq/react-no-ssr/blob/master/README.md) component to make sure it’s only rendered on client side.

**Comment by thierryc** [comment]
True, I already do NoSSR. 
```
const MyStageNoSSR = dynamic(() => import('./components/MyStage'), {
  ssr: false
});
```
This is not the origin of the issue.

Other prebuild components like works.

**Comment by inlet** [comment]
Hi @thierryc,

I've checked the create-react-app version, and you have misspelled the word `height` in jsx.

![image](https://user-images.githubusercontent.com/232559/49510982-22ee0680-f88a-11e8-8334-7f59c1c68db1.png)

Rename it to `height` and the red rectangle shows up

**Comment by inlet** [comment]
And tested it with NextJS, and it seems that this is causing the issue here:
https://github.com/zeit/next.js/issues/5254

TLDR; You need to move the `components` folder to the root and it works.

**Comment by inlet** [comment]
I'm closing this issue, please let me know if you still encounter issues and we can consider reopening it.

**Comment by thierryc** [comment]
Thank you
